mne/io/brainvision/brainvision.py

```diff
@@ -836,3 +836,16 @@ def read_raw_brainvision(vhdr_fname, montage=None,
     return RawBrainVision(vhdr_fname=vhdr_fname, montage=montage, eog=eog,
                           misc=misc, scale=scale, preload=preload,
                           stim_channel=stim_channel, verbose=verbose)
+
+
+def _bv_parser(description, add_for_r=1000):
+    """Parse BrainVision event codes (like `Stimulus/S 11`) to ints."""
+    try:
+        event_type, code = description[:-3], int(description[-3:])
+        if event_type in ("Response/R", "Stimulus/S"):
+            if event_type == "Response/R":
+                code += add_for_r
+            return code
+    except ValueError:  # probably not coercable to int
+        warn("Marker '%s' could not be unambiguously mapped to an int "
+             "and will be dropped." % description)
```

Comment:
Events not matching the pattern should probably not be dropped but assigned arbitrary integers (just like we did before). This will fix the current error in the test.

Reply:
we had that at some point and we reverted. (we can do it again)

Reply:
I think we have to, because we want to have all events, not just the ones matching the patterns.

Reply:
How should we do that? It must be deterministic, mapping the same code to the same int every time. We could do it externally, or hash it somehow, or use a decorator?

Reply:
Good question. Maybe sort the rest alphabetically and assign numbers starting with 2000?